### ISSUE TITLE
Code release

### ISSUE THREAD

**Issue by hongsukchoi** [nogu-atsu/NARF#1]
Hi, thanks for your great work.

Do you have a plan to release the code implementation of NARF?

**Comment by nogu-atsu** [comment]
Yes, we are planning to release the code in the future, but the schedule is not decided yet.

**Comment by hongsukchoi** [comment]
Could you give me an approximate schedule? I think the project is great and want to test several add-ons
